terraform/hcl2json.go

```diff
@@ -41,26 +41,6 @@ func ParseModule(rawFiles map[string]interface{}) map[string]interface{} {
 	}
 }
 
-// extractInputVariables extracts the input variables values from the provided file
-var extractInputVariables = func(file File) (ValueMap, error) {
```

Comment:
Refactored `extractInputVariables` and `extractLocals` into `ExtractVariables` under `variables.go`, since I don't think we should care in this file what type of variables/values we're extracting. These two functions looked more or less the same so they didn't need to be separate functions